src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+body,
+ul,
+li {
   margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  background-color: #fff;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
+  margin-top: 3px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav-links {
+  display: flex;
+}
+
+.link {
+  display: block;
+  text-decoration: none;
+  color: #000;
+  font-weight: 500;
+  position: relative;
+  font-size: 12px;
+  line-height: 60px;
+  height: 60px;
+  padding-right: 20px;
+}
+
+.link.is-active {
+  color: #00acdc;
+}
+
+.link.is-active::after {
+  display: block;
+  content: '';
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
```

Comment:
CSS Variable is used for a blue color
![image](https://github.com/user-attachments/assets/d5352e8f-a84a-4d4b-916b-120150ae3dea)